new-content/events/summer-school-2021.yaml

```diff
@@ -23,6 +25,8 @@ header:
     time:
     to: 'https://qiskit.org'
```

Comment:
Ask @JRussellHuffman if this link should have an arrow. If the answer is "yes", replace this url with "/"